examples/Huggingface_Transformers/Transformer_handler_generalized.py

```diff
@@ -228,7 +240,8 @@ def inference(self, input_batch):
         inferences = []
         # Handling inference for sequence_classification.
         if self.setup_config["mode"] == "sequence_classification":
-            predictions = self.model(input_ids_batch, attention_mask_batch)
+            with torch.inference_mode():
```

Comment:
Why not use `@torch.inference_mode()` decorator on `def inference()`

Reply:
done